gst/nnstreamer/include/tensor_typedef.h

```diff
@@ -136,6 +136,7 @@ typedef enum _nns_media_type
   _NNS_TEXT = 2, /**< supposedly text/x-raw */
   _NNS_OCTET = 3, /**< supposedly application/octet-stream */
   _NNS_MEDIA_PLUGINS = 0x1000, /**< external converters */
+  _NNS_MEDIA_ANY = 0x2000, /**< any media type */
```

Comment:
```
_NNS_MEDIA_ANY = 0x1000, /**< any media type (find proper external converter in tensor-converter element) */
```

IMO _NNS_MEDIA_PLUGINS as media type is weird, let's consider to change name and use external converter if mode is not custom.

Reply:
Agree. Changed!